isisd/isis_adjacency.c

```diff
@@ -814,6 +815,14 @@ void isis_adj_print_vty(struct isis_adjacency *adj, struct vty *vty,
 				vty_out(vty, "      %s\n", buf);
 			}
 		}
+		if (adj->circuit && adj->circuit->bfd_config.enabled) {
+			vty_out(vty, "    BFD is %s", adj->bfd_session ? "active" : "configured");
```

Comment:
move the \n to this vty_out and remove the else statement below?

Please also fix the style issues found